platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -137,6 +138,7 @@ private void handleMessage(final Message msg) {
 					}
 
 					int inputType = InputType.TYPE_CLASS_TEXT;
+					String accept_characters = "";
```

Comment:
```suggestion
					String acceptCharacters = "";
```
We use `camelCase` in java